### PR TITLE
Display products #1

### PR DESCRIPTION
**Pull request by clairefreehafer**
sample products rendering on the home page

### CODE REVIEW

File: app/reducers/product.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import axios from 'axios'
+
+export const initialState = {
+			name: 'Love Potion',
+			description: 'For falling in love.',
+			price: 500,
+			category: ['potion', 'love'],
+		}
```
Comment (rjacobskind) [inline app/reducers/product.jsx]: We made different category types -- we can change this easily though

File: app/reducers/products.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+export const loadProducts = function () {
+  return function (dispatch) {
+    fetch('/api/products')
+    .then(function (res) {
+			// console.log('products', res)
+      return res.json();
+    })
+    .then(function (products) {
+			console.log('products', products)
```
Comment (rjacobskind) [inline app/reducers/products.jsx]: console.log still necessary here?

File: app/reducers/products.jsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+      const action = getProducts(products); // this should dispatch to our store as set products to be all the products fetched via this api request
+      dispatch(action);
+    })
+    .catch(function (err) {
+      console.error(err)
+    });
+  };
+};
+
+export default function (state = [], action) {
```
Comment (rjacobskind) [inline app/reducers/products.jsx]: Is state meant to be set to an array here?

File: app/reducers/index.jsx [hunk 4]
```diff
@@ -1,7 +1,9 @@
 import { combineReducers } from 'redux'
 
 const rootReducer = combineReducers({
-  auth: require('./auth').default,  
+  auth: require('./auth').default,
+  products: require('./products').default,
+  product: require('./product').default,
 })
```
Comment (elliette) [inline app/reducers/index.jsx]: default is how they did it in Bones- don't know what that actually does but it seems to work

File: app/components/Product.jsx [hunk 5]
```diff
@@ -0,0 +1,8 @@
+import React, { Component } from 'react';
+
+export default function (singleProduct) {
+  console.log('props', singleProduct)
+  return (
+    <p>Hello world</p>
+  )
+}
```
Comment (elliette) [inline app/components/Product.jsx]: No we haven't started working on this yet- it is just a bare-bones outline

File: app/reducers/products.jsx (hunk 3 above)
Comment (elliette) [inline app/reducers/products.jsx]: Yeah, because we combine states later in our root reducer. This is only our products state.

**Pull request merged**